08-rails/exercises/erd-practice-activity.md

```diff
@@ -0,0 +1,21 @@
+# ERD Practice Activity
+
+## Scenario
+
+We are setting up an application to allow people to share their homes to tourists online like AirBnb.  So our initial user stories would be:
+
+- As a user I can add a home to be listed for rentals, along with a price, so I can rent the home out and make money.
+- As a user I can choose when a home of mine is available for rentals so I can pick when I can have guests.
+- As a user I can can select a home to rent for a vacation on a specific set of dates.
+- As a user I can see all the money I have spent.
```

Comment:
I'd group the top 3 bullets about setting up a rental together and then the bottom three together about reserving a rental